solr/solrj/src/test/org/apache/solr/client/solrj/response/TestSuggesterResponse.java

```diff
@@ -32,14 +33,15 @@
  * Test for SuggesterComponent's response in Solrj
  *
  */
-public class TestSuggesterResponse extends SolrJettyTestBase {
+public class TestSuggesterResponse extends EmbeddedSolrServerTestBase {
```

Comment:
[Q] Ditto re: changing base class to EmbeddedSolrServerTestBase

Reply:
Because the test uses the hidden EmbeddedSolrServer, so moved it to the new base